docs/sdk-and-tools/indices/miniblocks.md

```diff
@@ -0,0 +1,47 @@
+---
+id: es-index-miniblocks
+title: Elasticsearch index: miniblocks
+---
+
+The `_id` field for this index is composed of hex encoded miniblock hash.
```

Comment:
The `_id` field of this index is represented by the miniblock hash, in a hexadecimal encoding.

Reply:
done